swagger.yaml

```diff
@@ -26,14 +26,14 @@ securityDefinitions:
     name: X-Secret-Key
 
 paths:
-  /data/{device_id}/in:
+  /data/{deviceId}/in:
     get:
       operationId: GetDataMessageForDevice
       tags:
         - yggdrasil
       parameters:
         - in: path
-          name: device_id
+          name: deviceId
```

Comment:
Changes in swagger break compatibility with device-worker, which sends payload in that model.

Reply:
Are u saying I should not change swagger?

Reply:
Yes. Unless you can provide compatibility with the older model.